src/interfaces/utils-interfaces/document-interface.ts

```diff
@@ -0,0 +1,6 @@
+/** A document attached to a filing. */
+export interface DocumentIF {
```

Comment:
Lives in other projects as well, I took this from business-filings-ui

Reply:
Then should be moved to shared repo. Please add as a todo item to future ticket.

Reply:
New ticket is bcgov/entity#16489.